app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    num_of_errors = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            num_of_errors += 1
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+            num_of_errors += 1
+    if num_of_errors == 0:
+        return f"Friends can go to {cafe.name}"
```

Comment:
It's redundant condition and variable num_of_errors, make without it, left only masks_to_buy